fdbcli/fdbcli.actor.cpp

```diff
@@ -2156,13 +2156,6 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise, Reference<ClusterCo
 					continue;
 				}
 
-				if (tokencmp(tokens[0], "cache_range")) {
```

Comment:
If it never worked, then deleting user-facing feature w/o deprecation seems fine.